unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/SearchSubSection/SearchSubSectionComponentView.cs

```diff
@@ -14,8 +14,8 @@ public class SearchSubSectionComponentView : BaseComponentView, ISearchSubSectio
     public int CurrentTilesPerRow { get; }
     public int CurrentGoingTilesPerRow { get; }
 
-    [SerializeField] private GameObject minimalSearchSection;
-    [SerializeField] private GameObject fullSearchSection;
+    [SerializeField] internal GameObject minimalSearchSection;
+    [SerializeField] internal GameObject fullSearchSection;
```

Comment:
Why did you need to set these 2 variables as internal? I'm not seeing that you are using them in `SearchSubSectionComponentViewShould.cs`.